File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/non-plagiarized/04/T05.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function reverse(number: number): void {
  const cek: string = number.toString();
  let result: string = '';
  for (let i = cek.length - 1; i >= 0; i--) {
    result += cek[i];
  }
  console.log(result);
}

rl.question('Enter an integer ', (input) => {
  const a: number = parseInt(input);
  reverse(a);
  rl.close();
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/04/T05.py
import sys

def reverse(number):
    cek = str(number)
    for i in range(len(cek)):
        print(cek[len(cek)-1-i], end='')
    print()

if __name__ == '__main__':
    a = int(input("Enter an integer "))
    reverse(a)

